hr_attendance_overtime_negative_hours/readme/DESCRIPTION.md

```diff
@@ -0,0 +1 @@
+Override on `hr.leave.type:get_days_all_request()` to also show Negative Hours in Leaves overview (TimeOffCalendar widget)
```

Comment:
quite technical for a description: just keep `Show Negative Hours in Leaves overview (TimeOffCalendar widget)`

Reply:
I have updated it, thanks